namui-cli/src/main.rs

```diff
@@ -39,3 +51,58 @@ async fn main() {
         }
     }
 }
+
+#[derive(Clone, ArgEnum)]
+#[clap(short, long)]
+enum ElectronPackagePlatform {
```

Comment:
I think here are too many electron things in main.rs. please move it to proper file.